backend/app/schemacms/projects/admin.py

```diff
@@ -133,7 +133,10 @@ def save(self):
                     objs_with_deferred_fields.append(deserialized_object)
 
             for obj in objs_with_deferred_fields:
-                obj.save_deferred_fields()
+                try:
+                    obj.save_deferred_fields()
+                except Exception:
+                    continue
```

Comment:
I assume this won't throw any error or display any messages right?

Reply:
I dont think is necessary, this is like a second saving, for objects that needed to wait till another objects are created, so if something is missing is just because some old issues with migrations and it is not necessary